### ISSUE TITLE
Word API automatically inserts line break but only after the first Paragraph

### ISSUE THREAD

**Issue by algomaster99** [OfficeDev/office-js#1284]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

## Expected Behavior
The APIs should have either inserted a `LineBreak` after every heading or shouldn't have inserted a `LineBreak` at all

## Current Behavior
A `LineBreak` after first `Heading is inserted even though it isn't specified in the program.

## Steps to Reproduce, or Live Example

[Script Lab](https://gist.github.com/algomaster99/c942a984b24daaf542f9b411f559ad30)
1. Click `Run`
2. A `LineBreak` after the first Heading is inserted even though it isn't specified in the program.

## Context
I **need** to use `insertText`. I can't use `insertParagraph` in my application because I have to mark some content controls in the paragraph too so I render them word by word or sentence by sentence.

## Your Environment

- Platform [PC desktop, Mac, iOS, Office Online]: PC desktop
- Host [Excel, Word, PowerPoint, etc.]: Word
- Office version number: 18.2005.1191.0
- Operating System: Windows 10 Pro 1903

## Useful logs

- [ ] Console errors
- [x] Screenshots
![demo](https://user-images.githubusercontent.com/35191225/87776396-5609c180-c845-11ea-812a-1daf426529ff.PNG)
- [ ] Test file (if only happens on a particular file)

**Comment by algomaster99** [comment]
This doesn't happen on Word on the web though.
![demo-web](https://user-images.githubusercontent.com/35191225/87778515-f31a2980-c848-11ea-979b-bceb5456124a.PNG)
